### ISSUE TITLE
Feature Request - Print Entire Record

### ISSUE THREAD

**Issue by cjmny** [salesagility/SuiteCRM#1500]
Coming from MS CRM 3.0, one of the handiest features was the ability to print all tabs from an entire record (excluding one to many relationships/sub panels).  This was useful when reviewing items in any sort of meeting scenario.

**Comment by paulgodley** [comment]
Hi @cjmny, thank you for this, that is a very good suggestion.  @JimMackin can you please mark this call with the suggestion label?

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Feature Request - Print Entire Record](https://trello.com/c/OHUcnoBq/95-feature-request-print-entire-record)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
